arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -3010,6 +3010,11 @@ void RocksDBEngine::addSystemDatabase() {
   builder.add(StaticStrings::DatabaseName,
               VPackValue(StaticStrings::SystemDatabase));
   builder.add("deleted", VPackValue(false));
+  // Also store the ReplicationVersion when creating the Database
+  auto& df = server().getFeature<DatabaseFeature>();
+  builder.add(
+      StaticStrings::ReplicationVersion,
+      VPackValue(replication::versionToString(df.defaultReplicationVersion())));
```

Comment:
This is good.

I'm wondering, what are we doing on load, if the replication version isn't set? We must always default to replication version 1, and not to defaultReplicationVersion there - your comment on the system database breaking above suggest that's not the case. Can we double-check this, please?

Reply:
As far as I know, that is defaulted to version 1 (I checked it in [VocbaseInfo.h](https://github.com/arangodb/arangodb/blob/devel/arangod/VocBase/VocbaseInfo.h#L178)). But I'll also let Michael answer this one.